### PR TITLE
fix(es/module): preserve exports order

### PR DESCRIPTION
**Pull request by magic-akari**
**Description:**

I don't know if anyone will rely on export order. But preserving the order is easy for swc because we have collected all the span.

**BREAKING CHANGE:**

**Related issue (if exists):**
- https://github.com/babel/babel/issues/9177

### CODE REVIEW

File: crates/swc/tests/fixture/issues-2xxx/2964/case-2/output/index.ts [hunk 1]
```diff
@@ -10,8 +10,8 @@ function _export(target, all) {
 }
 _export(exports, {
     badIstanbul: ()=>badIstanbul,
```
Comment (magic-akari) [inline crates/swc/tests/fixture/issues-2xxx/2964/case-2/output/index.ts:12]: https://github.com/swc-project/swc/blob/428ca6a650cc221d017245568b71e43c7c440fca/crates/swc/tests/fixture/issues-2xxx/2964/case-2/input/index.ts#L1-L8

badIstanbul's span is lost. 

This issue will disappear if `...pixelParams` is removed.
I supposed it is a bug related to `object_rest_spread`.

Comment (magic-akari) [reply crates/swc/tests/fixture/issues-2xxx/2964/case-2/output/index.ts:12]: I will fix this soon in another pull request.

Comment (kdy1) [reply crates/swc/tests/fixture/issues-2xxx/2964/case-2/output/index.ts:12]: Cool, thanks!

**Review by kdy1 — APPROVED** [review]
Thanks!

---

swc-bump:
 - swc_ecma_transforms_module

**Comment by magic-akari** [comment]
It looks like something is wrong with setup-node.

**Pull request merged**